TrkReco/src/SimpleKalSeedSelector_tool.cc

```diff
@@ -0,0 +1,28 @@
+#include "Offline/TrkReco/inc/SimpleKalSeedSelector.hh"
+
+namespace mu2e {
+
+  bool SimpleKalSeedSelector::select(KalSeed const& kseed) const {
+    // evaluate the momentum at t0
+    if(kseed.intersections().size() > 0){
+      auto const& kinter = kseed.intersections().front();
+      auto mom = kinter.mom();
+      auto fcon = kseed.fitConsistency();
+      return mom >= minmom_ && mom <= maxmom_ && fcon >= minfcon_;
+    } else
+      return false;
+  }
+
+  bool SimpleKalSeedSelector::isBetter(KalSeed const& current,KalSeed const& test) const {
+    double nhitfrac = 2*(test.nHits() - current.nHits())/(test.nHits() + current.nHits());
+    if(nhitfrac > minsignhit_){
+      // difference is significant
```

Comment:
It's unclear to me what this is checking - it looks like its more than just whether one track has more hits than the other? A comment in the code to clarify would be good